### PR TITLE
Implement test WINFF.FT.S.SIQ.10

### PR DESCRIPTION
**Pull request by krlinus**
Missing highFrequency parameter in spectrum inquiry request
Old test case no. was - 6.2.4.2.1.2.2

### CODE REVIEW

**Comment by kate-harrison** [comment]
Janet and Carrie -- I just wanted to note that this PR is from Sunil, one of our people working on test script development. Please review it normally and let me know if you have any questions or concerns.

File: src/harness/testcases/spectrum_inquiry_testcase.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    spectrum_inquiry_0 = json.load(
+        open(os.path.join('testcases', 'testdata', 'spectrum_inquiry_0.json')))
+    spectrum_inquiry_0['cbsdId'] = cbsd_id
+    del spectrum_inquiry_0['inquiredSpectrum'][0]['highFrequency']
+    self.assertFalse(
+        'highFrequency' in spectrum_inquiry_0['inquiredSpectrum'][0])
+    request = {'spectrumInquiryRequest': [spectrum_inquiry_0]}
+    # Check Spectrum Inquiry Response
+    response = self._sas.SpectrumInquiry(request)['spectrumInquiryResponse'][0]
+    self.assertEqual(response['response']['responseCode'], 102)
```
Comment (qingwu123) [inline src/harness/testcases/spectrum_inquiry_testcase.py]: Need to check cbsdId is returned in the response.

File: src/harness/testcases/spectrum_inquiry_testcase.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    self.assertEqual(response['response']['responseCode'], 0)
+    cbsd_id = response['cbsdId']
+    del request, response
+
+    # Send Spectrum Inquiry request
+    spectrum_inquiry_0 = json.load(
+        open(os.path.join('testcases', 'testdata', 'spectrum_inquiry_0.json')))
+    spectrum_inquiry_0['cbsdId'] = cbsd_id
+    del spectrum_inquiry_0['inquiredSpectrum'][0]['highFrequency']
+    self.assertFalse(
```
Comment (qingwu123) [inline src/harness/testcases/spectrum_inquiry_testcase.py]: Is this necessary?

Comment (krlinus) [inline src/harness/testcases/spectrum_inquiry_testcase.py]: Removing this line.

File: src/harness/testcases/spectrum_inquiry_testcase.py (hunk 1 above)
Comment (krlinus) [inline src/harness/testcases/spectrum_inquiry_testcase.py]: Done

File: src/harness/testcases/spectrum_inquiry_testcase.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SIQ_10(self):
```
Comment (qingwu123) [inline src/harness/testcases/spectrum_inquiry_testcase.py]: Is it WINNF or WINFF?

Comment (krlinus) [inline src/harness/testcases/spectrum_inquiry_testcase.py]: WinnF = Wireless INNovation Forum.

Comment (krlinus) [inline src/harness/testcases/spectrum_inquiry_testcase.py]: On Pg 16 of the spec (version 5.5.1) this portion of the value can be either WINNF or FCC. I am guessing that the WINFF found later in that document is a typo, and will be updated at some time. I suggest we should go with what is correct, rather that following the name WINFF blindly. For now, I have changed it to WINFF, however, please let me know if we can agree on using WINNF.

**Pull request merged**